### ISSUE TITLE
Mob spawn config options

### ISSUE THREAD

**Issue by vgtgvgs** [Alex-the-666/Ice_and_Fire#1491]
This mod seems to break when paired with Open Terrain Generator (Biome Bundle), causing death worms to have extremely high spawn rates (even at min spawn rate and max spawn check). Tracking the exact source of the issue would be a headache, especially due to OTG's complexity.
Instead, would it be possible to add a config option to disable death worm spawns entirely? (and possibly one for all the other mobs, too)

**Comment by yqmtat** [comment]
have you tried to update llibrary?

**Comment by vgtgvgs** [comment]
yep, I'm using llibrary 1.7.18 (the March 10 release).
Additional info: I've got a bunch of other mods installed, but the death worm overspawning issue went away when I removed OTG, so I'm reasonably sure that's it. I've also read comments that OTG messes with mob spawns from other mods too, meaning the issue lies in that mod. However, resolving the issue on their side would likely take forever, meanwhile adding a config here would (presumably?) not be too long of a change.
